File: inclass/week1/iotest.py
# Ishaan is super cool!
# Dom is also very cool
# Maybe Brogan i dunno
# Brogan's okay I guess





line1 = input()
line1.split()
lines = int(line1[0])
divisor = int(line1[2])

count = 0
for _ in range(lines):
    if int(input()) % divisor == 0:
        count += 1
print(count)

File: inclass/week1/iotest.java
import java.io.*; 

public class iotest {

    public static void main(String[] args) throws IOException {
        BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
        String[] line1 = in.readLine().split(" ");
        int lines = Integer.parseUnsignedInt(line1[0]);
        int divisor = Integer.parseUnsignedInt(line1[1]);

        int total = 0;
        for(; lines > 0; lines--) {
            if (Integer.parseUnsignedInt(in.readLine()) % divisor == 0) {
                total++;
            }
        }
        System.out.println(total);
    }
}